### ISSUE TITLE
[Slider component] Right side of the slider has contrast ratio of 1.9:1. Need at least 3:1. 

### ISSUE THREAD

**Issue by vrgadekar** [microsoft/fast#2585]
**Describe the bug; what happened?**

**What are the steps to reproduce the issue?**

Example:

1. Go to reading mode (reading view) in Edge. For example - read://https_www.msn.com/?url=https%3A%2F%2Fwww.msn.com%2Fen-in%2Fnews%2Fother%2Findia-us-in-talks-for-president-trump-s-visit-to-india%2Far-BBYX7bO%3Focid%3Dmsedgntp

2. Click on 'Text preferences option'
3. Try the "Text size" slider

**The right half of slider should have contrast ratio of at least 3:1**

![image](https://user-images.githubusercontent.com/52927688/72403484-2f74ac00-3778-11ea-8c73-6a0f9293a6ca.png)

* OS & Device: [e.g. MacOS, Windows
* Browser [e.g. Edge]
* Latest Canary

**Comment by chrisdholt** [comment]
Pinging @bheston on this

**Comment by bheston** [comment]
This question has come up before, but as far as I understand this is not an accessibility requirement. The thumb is the indicator here, the difference in colors between the left and right side are not required for understanding or using the control. That mode is optional as it is.

**Comment by chrisdholt** [comment]
Closing based on https://github.com/microsoft/fast/issues/2585#issuecomment-575807431
